test/lib/ufe/testSelection.py

```diff
@@ -118,3 +119,26 @@ def snFront(sn):
             cmds.redo()
             self.assertEqual(len(globalSn), 1)
             self.assertEqual(snFront(globalSn), i)
+
+    def testUfeSelect(self):
+        def selectCmd(item):
+            sn = ufe.Selection()
+            sn.append(item)
+            ufeSelectCmd.replaceWith(sn)
+        self.runTestSelection(selectCmd)
+
+    @unittest.skipUnless(mayaUtils.previewReleaseVersion() >= 116, 'Requires Maya fixes only available in Maya Preview Release 116 or later.') 
+    def testMayaSelect(self):
```

Comment:
This test demonstrates the bug.  It would be desirable for this test to use the select command for all selections, but currently the select command cannot select using UFE path strings, which should be changed.  For now, it's sufficient to demonstrate the bug to use the select command to select Maya scene items, and the ufeSelectCmd to select non-Maya scene items.